variation-commons-beacon/src/main/java/uk/ac/ebi/eva/commons/beacon/models/BeaconAlleleRequest.java

```diff
@@ -52,9 +52,9 @@
 
     private String includeDatasetResponses;
 
-    public BeaconAlleleRequestBody() { }
+    public BeaconAlleleRequest() { }
 
-    public BeaconAlleleRequestBody(String referenceName, Long start, Long startMin, Long startMax, Long end,
+    public BeaconAlleleRequest(String referenceName, Long start, Long startMin, Long startMax, Long end,
                                    Long endMin, Long endMax, String referenceBases, String alternateBases,
```

Comment:
Fix indentation of the lines listing parameters.

Reply:
Fixed :)